src/reducers/inputs.js

```diff
@@ -4,6 +4,16 @@ import Validator from '../Validator'
 
 export const validate = (state, action = { payload: {} }) => new Validator(state).validate(action.payload.show)
 
+/**
+ *  @param {Object} action the payload of the action is a boolean that says to show or not to show the chooser
+ */
+export const showGroupChooser = (state, action) => {
```

Comment:
need test for this, no?

Reply:
We haven't added any functionality.  If this function was adequately tested before, why do we need more tests now?

Reply:
It seems adding a function implies there should be tests for the function, in most cases.  If it wasn't adequately tested before ... isn't now the time to fix that?

Reply:
that seems like a good idea (and a thing we've been doing elsewhere, right?).  no pref from me as to whether that happens in this PR or a follow on.

Reply:
if the current test is at the component level, but the component functionality is now wired differently (managing its state via redux) than before... and if there's no higher level integration test, is there anything actually testing the redux wiring?  that is to say, the previously adequate testing no longer seems as thorough with the new implementation, right?  i would expect this to be the reason for the small test coverage drop?  or am i missing how the new reducer gets exercised?

Reply:
Okay, test added.  @jmartin-sul yes, the integration tests cover this, but we're just not recording the data from that. We're only doing coverage of unit tests.   I think unit tests for simple operations like this are not very valuable.

Reply:
ah, yeah, ok.  sorry, didn't realize there was an integration test for this.